tendrl/commons/utils/ssh/authorize_key.py

```diff
@@ -34,19 +35,33 @@ def run(self):
         output:
             True/False, error
         """
+        if self.attributes["user"] is 'root':
+            try:
+                os.mkdir("/root/.ssh", 700)
+            except OSError:
+                pass
+            open('/root/.ssh/authorized_keys', 'a').close()
```

Comment:
@r0h4n Because if authorized_keys file is not present then first run it is creating authorized_keys file, but key is not copied, next time run it copying key inside authorize_key file, so i am creating authorize_key file if it is not present, so in first run copy key will done successfully